lib/setupHooks/removeReferencesToVendoredSources.nix

```diff
@@ -22,10 +22,10 @@ makeSetupHook
         (
           exec 3>&1
           echo signing files:
-          find "''${installLocation}" -type f |
-            sort |
-            tee -a /dev/fd/3 |
-            xargs --no-run-if-empty signIfRequired
+          while IFS= read -r -d $'\0' file; do
+            echo "signing: $file"
+            signIfRequired "$file"
+          done < <(find "''${installLocation}" -type f -print0)
```

Comment:
Note: Files are no longer sorted, but that's probably fine?

Reply:
Does it break determinism in any way?

Reply:
It shouldn't, because all the same paths get signed, unless you mean determinism of log output?

Reply:
Just double checking. The determinism of log output doesn't matter, AFAIK.

Reply:
I originally added the sort in there just to keep stuff deterministic in the logs, but it's fine to drop (we can always revisit if there is something more to debug)